errcheck/errcheck.go

```diff
@@ -335,6 +336,34 @@ func (v *visitor) fullName(call *ast.CallExpr) string {
 	return fn.FullName()
 }
 
+func getSelectorName(sel *ast.SelectorExpr) string {
+	if s, ok := sel.X.(*ast.SelectorExpr); ok {
+		return fmt.Sprintf("%s.%s", getSelectorName(s), sel.Sel.Name)
+	}
+
+	return ""
+}
+
+// fullName will return a package / receiver-type qualified name for a called function
```

Comment:
this documentation appears to be incorrect

Reply:
forgot to commit the changes fro documentation